src/main/resources/com/puppycrawl/tools/checkstyle/packages_1_1.dtd

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!ELEMENT checkstyle-packages (package)*>
+
+<!ELEMENT package (package)*>
+<!ATTLIST package name NMTOKEN #REQUIRED>
```

Comment:
please recheck why Checkstyle is not catching this as violation. We enforce all files to have newline at the end